diff --git a/python/sglang/srt/utils/offloader.py b/python/sglang/srt/utils/offloader.py
index e38b0f2a19..bf1293bf2e 100644
--- a/python/sglang/srt/utils/offloader.py
+++ b/python/sglang/srt/utils/offloader.py
@@ -145,7 +145,13 @@ class OffloaderV1(BaseOffloader):
                     k: v.to(device, non_blocking=True)
                     for k, v in module.state_dict().items()
                 }
-                output = functional_call(module, device_state, args=args, kwargs=kwargs)
+                output = functional_call(
+                    module,
+                    device_state,
+                    args=args,
+                    kwargs=kwargs,
+                    tie_weights=False,
+                )
                 module.forward = forward
                 return output
 
